test/adexp_neuron.jl

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+adparam = SNN.ADEXParameter(;a = 6.0,
+    b = 7.3,
+    cm = 803.1,
+    v_rest= -63.2,
+    tau_m=19.7,
+    tau_w=351.0,
+    v_thresh=-39.2,
+    delta_T=6.4,
+    v_spike = -36.6,
+    v_reset = -59.2,
+    spike_delta = 16.3)
+
+E = SNN.AD(;N = 1, param=adparam)
+E.I = [795.57128906]
+SNN.monitor(E, [:v,:I])
+SNN.sim!([E],[], dt=0.1ms, duration=2000ms)
+# # Julia SNN Implementation of AdExp Neuron.
```

Comment:
Please move this reference to `src/neuron/adexp.jl`

Reply:
I wonder if the wrong code is highlighted here and you mean move the model reference text to src/neuron/adexp.jl?  

I ask because I have seen a similar pattern with other model tests where the models capacity to stimulate and record is tested. For example [here](https://github.com/AStupidBear/SpikingNeuralNetworks.jl/blob/master/test/hh_neuron.jl#L4-L5)

Reply:
I mean that the reference to the source paper should go together with its implementation, not with the tests (where curious users may not see it when browsing the code).